helix-view/src/document.rs

```diff
@@ -544,6 +552,7 @@ impl Document {
         &mut self,
         path: Option<PathBuf>,
         force: bool,
+        config: &Config,
```

Comment:
the extra argument is not necessary here. Since #5420 is merged you can retrieve the config from `self.config.load()` directly on the document (it's all synchronized automatically)